app/partials/app.cjsx

```diff
@@ -11,7 +11,8 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
 
   @tokens = ['zooHome', 'zooTalk', 'zooniverse/gravity-spy']
```

Comment:
Yeah, this is a filter to log Gravity Spy. I have no better ideas for filtering logging so that we aren't recording data for every project.